test/suites/ACID.m

```diff
@@ -2742,3 +2747,109 @@
   xlim([-1, 2]);
 end
 % =========================================================================
+function [stat] = colorbars()
+  stat.description = 'Manual positioning of colorbars';
+  stat.issues      = [933 937];
+
+  shift = [0.2 0.8 0.2 0.8];
+  axLoc = {'in','out','out','in'};
+
+  for iAx = 1:4
+    hAx(iAx) = subplot(2,2,iAx);
+    axPos    = get(hAx(iAx), 'Position');
+    cbPos    = [axPos(1)+shift(iAx)*axPos(3), axPos(2), 0.02, 0.2]; 
+
+    hCb(iAx) = colorbar('Position', cbPos, 'AxisLocation', axLoc{iAx});
+    title(sprintf('AxisLocation = %s', axLoc{iAx}));
+    grid('on');
+  end
+end
+% =========================================================================
+function [stat] = colorbarManualLocationRightOut()
+  stat.description = 'Manual positioning of colorbars';
+  stat.issues      = [933 937];
+
+  axLoc = 'out'; 
+
+  figure('Units','centimeters','Position',[1,1,11,10]);
+  axPos(1,:) = [1,1,8,3];
+  axPos(2,:) = [1,5,8,3];
+  cbPos      = [9.5,1,0.5,7]; 
+
+  hAx(1) = axes('Units','centimeters','Position',axPos(1,:));
+  imagesc([1,2,3],[4,5,6],rand(3),[0,1]);
```

Comment:
It is a bad practice to use `rand` inside testing (don't worry, I will fix it), since the output can change during each run. Instead, functions like `magic` can come in handy to have deterministic data.